app/uk/gov/hmrc/nisp/controllers/NIRecordController.scala

```diff
@@ -114,7 +109,7 @@ trait NIRecordController extends NispFrontendController with PertaxHelper {
 
       val nationalInsuranceResponseF = nationalInsuranceService.getSummary(nino)
       val statePensionResponseF = statePensionService.getSummary(nino)
-      (for (
+      for (
         nationalInsuranceRecordResponse <- nationalInsuranceResponseF;
```

Comment:
It was required because parenthesis was being used to to wrap the for expression statements. I have updated this to use curly braces as most people are familiar with this approach and you don't need the semi colon.